docs/remote/tailscale/basic.md

```diff
@@ -0,0 +1,65 @@
+# Basic Tailscale Setup
+
+These instructions will help you create a Tailnet and connect an Unraid server. Once finished, you should be able to
+connect to the following services via the Tailscale IP / MagicDNS name of the Unraid server:
+
+- WebGUI
+- File shares (SMB/NFS)
+- SSH
+- Bridge mode Docker containers (default or custom bridge networks)
+
+!!! note
+    Docker containers running on ipvlan/macvlan networks (e.g., br0) cannot be accessed via the Tailscale IP for the
+    Unraid server. To connect to these containers, you should either [configure a subnet router](advanced.md) or 
```

Comment:
```suggestion
    Unraid server. To connect to these containers, you should either [configure a subnet router](advanced.md#subnet-router) or 
```

Reply:
I intentionally linked to the top of the doc there, because getting to the point where you can change the subnet router settings is a little odd (which is what the first part of the page works through).

This might also be moot... I wasn't thinking when I started writing over here, but since Tailscale is becoming an official plugin these guides might also end up on the official docs site. I'll have to check on that part :)

Reply:
Im fine with the docs living here until that happens. It probably also wouldn't hurt if the docs can be in unraids docs repo before it is in, as long as you can make it clear its not in yet.

Reply:
I talked to folks and the preference is to keep the official docs simpler, so this works well here :)